### ISSUE TITLE
There's something missing to make it work. 

### ISSUE THREAD

**Issue by uyriq** [danstepanov/create-expo-stack#95]
created with 
# pnpm dlx create-expo-stack@latest
with some errors info 
```
> LittleLEmonExpoDan@1.0.0 format D:\proj\LittleLEmonExpoDan
> eslint '**/*.{js,jsx,ts,tsx}' --fix && prettier '**/*.{js,jsx,ts,tsx,json}' --write


Oops! Something went wrong! :(

ESLint: 8.55.0

No files matching the pattern "'**/*.{js,jsx,ts,tsx}'" were found.
Please check for typing mistakes in the pattern.

 ELIFECYCLE  Command failed with exit code 2.

```
then message Success! 🎉 Now, just run the following to get started
 
```
Your project configuration:
{
  projectName: 'LittleLEmonExpoDan',
  packages: [
    {
      name: 'react-navigation',
      type: 'navigation',
      options: { type: 'stack' }
    },
    { name: 'nativewind', type: 'styling' }
  ],
  flags: { noGit: undefined, noInstall: undefined, importAlias: '~/' }
}

To recreate this project, run:
  npx create-expo-stack@latest LittleLEmonExpoDan --react-navigation --nativewind --pnpm

```
# then `pnpm run android`

```
› Opening on Android...
› Opening exp://192.168.1.2:8081 on Medium_Phone_API_34
› Press ? │ show all commands
Error: Unable to resolve module ./node_modules\expo\AppEntry from D:\proj\LittleLEmonExpoDan/.:

None of these files exist:
  * node_modules\expo\AppEntry(.native|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx|.android.js|.native.js|.js|.android.jsx|.native.jsx|.jsx|.android.json|.native.json|.json|.android.cjs|.native.cjs|.cjs)
  * node_modules\expo\AppEntry\index(.native|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx|.android.js|.native.js|.js|.android.jsx|.native.jsx|.jsx|.android.json|.native.json|.json|.android.cjs|.native.cjs|.cjs)
    at ModuleResolver.resolveDependency (D:\proj\LittleLEmonExpoDan\node_modules\.pnpm\metro@0.76.8\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:114:15)
    at DependencyGraph.resolveDependency (D:\proj\LittleLEmonExpoDan\node_modules\.pnpm\metro@0.76.8\node_modules\metro\src\node-haste\DependencyGraph.js:277:43)
    at D:\proj\LittleLEmonExpoDan\node_modules\.pnpm\metro@0.76.8\node_modules\metro\src\lib\transformHelpers.js:169:21
    at Server._resolveRelativePath (D:\proj\LittleLEmonExpoDan\node_modules\.pnpm\metro@0.76.8\node_modules\metro\src\Server.js:1045:12)
    at async Server.requestProcessor [as _processBundleRequest] (D:\proj\LittleLEmonExpoDan\node_modules\.pnpm\metro@0.76.8\node_modules\metro\src\Server.js:449:37)
    at async Server._processRequest (D:\proj\LittleLEmonExpoDan\node_modules\.pnpm\metro@0.76.8\node_modules\metro\src\Server.js:383:7)
Error: Unable to resolve module ./node_modules\expo\AppEntry from D:\proj\LittleLEmonExpoDan/.:
```
// package.json
```package.json
{
  "name": "LittleLEmonExpoDan",
  "version": "1.0.0",
  "scripts": {
    "android": "expo start --android",
	
    	"format": "eslint '**/*.{js,jsx,ts,tsx}' --fix && prettier '**/*.{js,jsx,ts,tsx,json}' --write",
	
    "ios": "expo start --ios",
    "start": "expo start",
    "web": "expo start --web"
  },
  "dependencies": {
    
      "nativewind": "^2.0.11",
    

    

    
      "@react-navigation/native": "^6.1.7",
      "react-native-gesture-handler": "~2.12.0",
      "react-native-safe-area-context": "^4.6.3",
      "react-native-screens": "~3.22.0",
    

    
      "@react-navigation/stack": "^6.3.17",
    

    

    

    
    
    

	

	

    "expo": "~49.0.11",
    "expo-status-bar": "~1.6.0",
    "react": "18.2.0",
    "react-native": "0.72.6"
  },
  "devDependencies": {
    "@babel/core": "^7.20.0",
    "@types/react": "~18.2.14",
    "@typescript-eslint/eslint-plugin": "^6.7.2",
    "@typescript-eslint/parser": "^6.7.2",
    "eslint": "^8.50.0",
    "eslint-config-universe": "^12.0.0",
    "prettier": "^3.0.3",
    
      "tailwindcss": "3.3.2",
    
    
    "typescript": "^5.1.3"
  },
  "eslintConfig": {
    "extends": "universe/native"
  },
  
    "main": "node_modules/expo/AppEntry.js",
  
  
    "expo": {
      "install": {
        "exclude": ["react-native-safe-area-context"]
      }
    },
  
  "private": true
}
```

**Comment by danstepanov** [comment]
@frankcalise mind digging into this as it's a case of android use?

@uyriq I notice you're using pnpm dlx rather than npx, I can't guarantee that it will work with create-expo-stack as we haven't tested it with that

**Comment by uyriq** [comment]
I solved my problem, I'll document solution for future reference, maybe this will be useful for a similar FAQ question, etc. The cause of the error, as you noticed, was not fully configured _npm_ (I had an error when executing `npm root -g`). Since I was using _pnpm_ all the time, I didn't pay attention to the fact that it needed to be configured well with

- `npm config --global set prefix "<npm pkgs location>"`
- `npm config --global set cache "<npm cache location>"`

and that the path to the npm packages should be added to the path variable
additionally, following SO's advice, I added the ~/.npmrc file with the same prefix and cache values 
As a result, both ways of creating a project with `create-expo-stack` as with 

- `npx create-expo-stack@latest my-expo-npm-app` and
- `pnpm dlx create-expo-stack@latest my-expo-app`

Work well. Then this issue is solved.
